README.md

```diff
@@ -184,7 +184,7 @@ import React from 'react'
 
 export default class Error extends React.Component {
   static getInitialProps ({ res, xhr }) {
-    const statusCode = res ? res.statusCode : xhr.status
+    const statusCode = res ? res.statusCode : (xhr ? xhr.status : 500)
```

Comment:
There is a case no `req` and no `xhr` since `_error` component is sometimes used when runtime errors occurred on client. Though it's a bit weird to say `500` on this situation.